torchsynth/module.py

```diff
@@ -354,7 +354,12 @@ def __init__(
         # TODO: Make sure this is on GPU
         # https://pytorch-lightning.readthedocs.io/en/latest/advanced/multi_gpu.html#init-tensors-using-type-as-and-register-buffer
         # Do we want to detach clone?
-        self.phase = self.get_parameter("initial_phase").detach().clone()
+        # Do we want this persistent?
```

Comment:
So persistence would store this in the state_dict -- what would be the benefit of that? Saving and loading a synth? I don't think that keeping these values is important as long as it is always initialized with the initialize phase parameter values.